packages/fast-components-react-base/src/tabs/README.md

```diff
@@ -1,5 +1,8 @@
 # Tabs
 Use *tabs* when there is a paged interface where one controlling tab element corresponds to one tab panel.
 
+## Usage
+The *tabs* component can be passed a collection of *tabitems* via the `tabitems` prop. `tabitems` expects an array of the interface `TabItems`. `Tabitems` consists of `tab`, `content` and `id`. `tab` and `content` will render a ReactNode and `id` is a string that represents its unique identifier.
```

Comment:
The casing is inconsistent, when referring to the prop it should be `tabItems`, when referring to the component it should be `TabItem`.